Source/Core/Core/HW/EXI/EXI_DeviceMemoryCard.cpp

```diff
@@ -158,10 +159,33 @@ CEXIMemoryCard::CEXIMemoryCard(const int index, bool gciFolder) : card_index(ind
   SetCardFlashID(header.data(), card_index);
 }
 
-void CEXIMemoryCard::SetupGciFolder(u16 sizeMb)
+std::pair<std::string /* path */, bool /* migrate */>
+CEXIMemoryCard::GetGCIFolderPath(int card_index, bool allow_movie_folder)
 {
+  const std::string path_override =
```

Comment:
```suggestion
  std::string path_override =
```

(required to avoid copying `path_override`)